src/scripts/scripts/zone/caverns_of_time/old_hillsbrad/boss_captain_skarloc.cpp

```diff
@@ -287,7 +287,7 @@ struct boss_captain_skarlocAI : public ScriptedAI
         else
             DevotionAura_Timer -= diff;
 
-        if (HeroicMode)
+        //Consecration (Heroic)
```

Comment:
Also you should consider running a script that changes every < diff in the repo.